erpnext/projects/doctype/task/task.py

```diff
@@ -202,7 +202,7 @@ def set_tasks_as_overdue():
 	tasks = frappe.get_all("Task", filters={'status':['not in',['Cancelled', 'Closed']]})
```

Comment:
```suggestion
	tasks = frappe.get_all("Task", filters={'status':['not in',['Cancelled', 'Closed']]}, fields=["name", "review_date"])
```
It doesn't make sense to `frappe.db.get_value` when you can get all the values in `frappe.get_all`

Reply:
I could make this change but review date is not nessecary for most. In a test just now a standard query with this change is 1.2ms slower then the current setup. Should i make the change ?